server/config/config.go

```diff
@@ -195,6 +197,7 @@ func NewConfig() *Config {
 	fs.StringVar(&cfg.Security.KeyPath, "key", "", "path of file that contains X509 key in PEM format")
 	fs.BoolVar(&cfg.ForceNewCluster, "force-new-cluster", false, "force to create a new one-member cluster")
 
+	cfg.EnableServiceMiddleware = true
```

Comment:
I prefer to move it to another place and change the default value to false.

Reply:
Currently can remove it and the default value will be false